source/lib/course-search/types.js

```diff
@@ -6,13 +6,16 @@ export type CourseType = {
 	crsid: number,
 	departments: string[],
 	description?: string[],
+	enroll: number,
 	gereqs?: string[],
 	instructors: string[],
 	level: number,
 	locations?: string[],
+	max: number,
 	name: string,
 	notes?: string[],
 	number: number,
+	open: boolean,
```

Comment:
"Open" is different from "full", though… `status: open` is open (vs closed or cancelled); being "full" is different from being "closed".

A professor can let more people into a "full" class, for instance. (Honestly, I don't quite know what `status: closed` _means_, but it's different from being "full".)

Would you change this to `full` or something, please?